react-app/src/Mine/index.js

```diff
@@ -7,14 +7,40 @@ import TodaySummary from './TodaySummary'
 import Setting from '../Setting'
 import WeekCalendar from './WeekCalendar'
 
-export default function Mine({ db, setReset, reset }) {
+export default function Mine({ db, setReset, reset, exercises }) {
+  const [todayRecords, setTodayRecords] = useState([])
+  const [allRecords, setAllRecords] = useState([])
+  const [startDate, setStartDate] = useState('')
+  const toYYYYMMDD = (timestamp) => {
+    new Date(timestamp).toISOString().split('T')[0]
```

Comment:
Do we need return here?

Reply:
Oh yep, I think should add a `return` in front of `new Date…`